### DISCUSSION TITLE
How to lock down a certain tier of a resource

### DISCUSSION THREAD

**Discussion by chanakanissanka** [gettek/terraform-azurerm-policy-as-code#44] [Q&A] [answered]
Hi there, 
Like to ask the best possible way with this set-up to lock down certain resources. In my case, I want to stop people deploying APIM on the premium tier.

**Comment by gettek** [comment, marked as answer]
Hi @chanakanissanka, if you mean restricting certain resource types then please give this Policy a try: [deny_resources_types.json](https://github.com/gettek/terraform-azurerm-policy-as-code/blob/main/policies/General/deny_resources_types.json)

You can also do the opposite rule with: [whitelist_resources.json](https://github.com/gettek/terraform-azurerm-policy-as-code/blob/main/policies/General/whitelist_resources.json)
